ms/MSOper/MSMetaData.cc

```diff
@@ -3878,17 +3878,23 @@ std::pair<MDirection, MDirection> MSMetaData::getPointingDirection(
 		row >= this->nRows(),
 		"Row number exceeds number of rows in the MS"
 	);
-	SHARED_PTR<Vector<Int> > ant1, ant2;
-	_getAntennas(ant1, ant2);
-	antenna1 = (*ant1)[row];
-	antenna2 = (*ant2)[row];
-	time = (*_getTimes())[row];
+	// KS note: 2015/10/30
+	// getColum method (also used in _getAntennas and _getTimes) is
+	// too expesive for the current purpose of getting single cell.
+	String ant1ColName = MeasurementSet::columnName(MSMainEnums::ANTENNA1);
+	String ant2ColName = MeasurementSet::columnName(MSMainEnums::ANTENNA2);
```

Comment:
probably its not significant but I guess these could be const references instead of copies.